Source/WebCore/platform/Scrollbar.cpp

```diff
@@ -478,4 +478,14 @@ bool Scrollbar::supportsUpdateOnSecondaryThread() const
 #endif
 }
 
+ScrollbarHidden scrollbarHidden(Scrollbar* scrollbar)
```

Comment:
Make a static function on Scrollbar. Argument can be `const Scrollbar*`

Reply:
Fixed.